src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsEvolveObject.java

```diff
@@ -47,16 +44,9 @@ public void verifyEvolveCall() {
     verifyNoEvolveInSlot(AdsContent.PREFOOTER_RIGHT);
   }
 
-  public void verifyEvolveHopInSlot(String slotName) {
-    WebElement adSlot = driver.findElement(By.id(slotName));
-    java.util.List<WebElement> providers = adSlot.findElements(By.tagName("div"));
-    WebElement evolveDiv = providers.get(0);
-    Boolean evolveIframePresent = adSlot.findElements(
-        By.cssSelector("iframe[src*=\'" + EVOLVE_DFP_URL + "\']")).size() > 0;
-    assertFalse(evolveIframePresent);
-    WebElement evolveHopScript = evolveDiv.findElement(
-        By.cssSelector("script[type='text/javascript']"));
-    String evolveHopMessage = String.format("window.evolve_hop('%s')", slotName);
-    assertStringContains(evolveHopMessage, evolveHopScript.getAttribute("src"));
+  public void verifyEvolveHoppedInSlot(String slotName, String nextProviderSrc) {
+    String nextProviderSlotCSS = String.format("div[id*=\'%s/%s\']", nextProviderSrc, slotName);
```

Comment:
I'm going to nitpick a little bit ;) Consider going with camel-case: `nextProviderSlotCss`